imapserver/search.go

```diff
@@ -328,6 +328,37 @@ func readSearchKeyWithAtom(criteria *imap.SearchCriteria, dec *imapwire.Decoder,
 		criteria.Or = append(criteria.Or, or)
 	case "$":
 		criteria.UID = append(criteria.UID, imap.SearchRes())
+	case "MODSEQ":
+		if !dec.ExpectSP() {
+			return dec.Err()
+		}
+		var quotedName, name string
```

Comment:
I don't think we need two variables for the name here? A single should be enough?

If `Quoted` doesn't find a quoted string, it leaves the pointer unchanged.